Update 2024-02-29-third-party-cookies.md (#1535)
oktadeveloper/okta-blog: _source/_posts/2024-02-29-third-party-cookies.md

type: awareness
---

> **NOTE**: In July, Google provided an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/). Okta will work with Google to understand the specific impact on Okta customers. In the meantime, we continue to advise customers to migrate away from using 3rd party cookies, as outlined below.

Author: ```suggestion
**NOTE**: In July, Google provided an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/). Okta will work with Google to understand the specific impact on Okta customers. In the meantime, we continue to advise customers to migrate away from using third-party cookies, as outlined below.
```

> **NOTE**: In July, Google has given an update on their [3rd party cookie deprecation effort](https://developers.google.com/privacy-sandbox/cookies).  Okta will be working with Google to understand the specific impact to Okta customers.  In the meantime, we continue to advise customers to migrate way from the use of 3rd party cookies, as outlined below

Reviewer: Please ensure "third-party cookie" is used consistently.

Reviewer: Also ensure punctuation (missing period).

> **NOTE**: In July, Google has given an update on their [3rd party cookie deprecation effort](https://developers.google.com/privacy-sandbox/cookies).  Okta will be working with Google to understand the specific impact to Okta customers.  In the meantime, we continue to advise customers to migrate way from the use of 3rd party cookies, as outlined below

Author: ```suggestion
**NOTE**: In July, Google has given an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/).  Okta will be working with Google to understand the specific impact to Okta customers.  In the meantime, we continue to advise customers to migrate way from the use of 3rd party cookies, as outlined below.
```

> **NOTE**: In July, Google has given an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/).  Okta will be working with Google to understand the specific impact to Okta customers.  In the meantime, we continue to advise customers to migrate way from the use of 3rd party cookies, as outlined below.

Reviewer: ```suggestion
**NOTE**: In July, Google provided an update on their [third-party cookie deprecation effort](https://privacysandbox.com/news/privacy-sandbox-update/). Okta will work with Google to understand the specific impact on Okta customers. In the meantime, we continue to advise customers to migrate away from using 3rd party cookies, as outlined below.
```
